lib/commands/exam.js

```diff
@@ -176,21 +241,17 @@ module.exports = TestCommand.extend({
   },
 
   /**
-   * Generates the test pages to use for parallel runs. For a given baseUrl,
-   * it appends the total split count and the partition numbers each page is
-   * running as query params.
+   * Generates multiple test pages with browser ids.
```

Comment:
I do think the original description did a pretty good job at describing. 
how about we modify this to:
```
Generates multiple test pages: for a given baseUrl, it appends the partition numbers or the browserId each page is running as query params.
```